File: Basic_Concepts/JS/pattern4.js
// Question 3 :  PRINT THE BELOW PATTERN

// 1
// 1 2
// 1 2 3
// 1 2 3 4
// 1 2 3 4 5

for (let row = 1; row <= 5; row++) {
  let line = "";
  for (let col = 1; col <= row; col++) {
    line+=col+" "
  }
  console.log(line.trim())
}

File: Basic_Concepts/CPP/Pattern_Problems/pattern4.cpp
// Question 4 :  PRINT THE BELOW PATTERN

// 1
// 1 2
// 1 2 3
// 1 2 3 4
// 1 2 3 4 5

#include <iostream>
using namespace std;
int main()
{
    for (int row = 1; row <= 5; row++)
    {
        for (int col = 1; col <= row; col++)
        {
            cout << col << " ";
        }
        cout<<endl;
    }
}